zeebe/exporters/camunda-exporter/src/main/java/io/camunda/exporter/schema/SchemaManager.java

```diff
@@ -73,6 +75,40 @@ public void startup() {
     LOG.info("Schema management completed.");
   }
 
+  private void updateSchemaSettings() {
+    final var existingTemplateNames =
+        searchEngineClient
+            .getMappings(config.getIndex().getPrefix() + "*", MappingSource.INDEX_TEMPLATE)
+            .keySet();
+
+    final var existingIndexNames =
+        searchEngineClient.getMappings(allIndexNames(), MappingSource.INDEX).keySet();
+
+    indexTemplateDescriptors.stream()
+        .filter(desc -> existingTemplateNames.contains(desc.getTemplateName()))
+        .forEach(
+            desc -> {
+              searchEngineClient.updateIndexTemplateSettings(
+                  desc, getIndexSettings(desc.getIndexName()));
```

Comment:
🔧 Just a nitpick: Maybe we call it `getIndexSettingsFromConfig` or something, because for me it was not clear where these settings come from and whether it is not from ES or config, etc.